0004-ledger-plugin-interface/0004-ledger-plugin-interface.md

```diff
@@ -236,6 +237,35 @@ p.send({
 
 For a detailed description of these properties, please see [`OutgoingTransfer`](#outgoingtransfer).
 
+#### sendMessage
+<code>ledgerPlugin.sendMessage( **message**:[OutgoingMessage](#outgoingmessage) ) ⇒ Promise.&lt;null></code>
```

Comment:
I'll add an explanation of the feature's purpose.
If the underlying ledger doesn't directly support messaging it can use zero-amount transfers, right?